### ISSUE TITLE
node-red integration

### ISSUE THREAD

**Issue by moshed** [whatadewitt/yahoo-fantasy-sports-api#74]
any chance you would look into integrating this into node-red? im a bit of a beginner and the visual aspect of node-red really helps me visualize writing flows to accomplish certain things.

**Comment by whatadewitt** [comment]
I've never heard of this but happy to look into it when time allows. Might be a great opportunity to learn something yourself though :)

**Comment by whatadewitt** [comment]
Node-Red is equal parts cool, scary, and blasphemous... I'll dig deeper into the docs at some point and see if I can find a way to integrate anything.

In the meantime, you could check out how I set up the docs site https://github.com/whatadewitt/fantasy-docs-next which, outside of the catch all API route for all of the actual API calls, should provide a pretty simple introduction to how to write NodeJS code for the library.

**Comment by moshed** [comment]
yeah node red is great I use it to implement simple speech to text voice transcription bots in telegram 
![image](https://user-images.githubusercontent.com/8683379/116653141-9493e700-a954-11eb-821e-e2bf1b0f440a.png)

https://user-images.githubusercontent.com/8683379/116653453-36b3cf00-a955-11eb-8b07-bf4f8dead42c.MP4

control the lights in my house based on sunset and sunup and send notification to pushover about events.

![image](https://user-images.githubusercontent.com/8683379/116653533-66fb6d80-a955-11eb-9611-5fadba38263d.png)

and posts videos of all MLB homers to twitter 
![image](https://user-images.githubusercontent.com/8683379/116653652-a629be80-a955-11eb-86e6-0cc755e1ee29.png)

im interested in pulling my team roster from yahoo and posting the results of every at bat or inning pitched to my phone 

ill check the docs you linked! seems like a great place to start

**Comment by moshed** [comment]
Hey did you any luck integrating into node-red I was wholly unsuccessful in even the most basic steps of accessing the api and I'm afraid it's beyond my abilities. 

Thanks for all your hard work!

**Comment by whatadewitt** [comment]
I've been so busy man, I apologize.

Will let you know when I get a chance to look at it!

**Comment by moshed** [comment]
no need to apologize at all! whenever you get a shot that'd be great, if not, not. No need to apologize!
